.github/workflows/doc_tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Doc Tests
+on:
+  push:
+  pull_request:
+    branches:
+      - develop
+      - 'release/*'
+  workflow_dispatch:
+    branches:
+      - 'feature/doc-actions'
```

Comment:
Before this work get's merged, these two lines should be removed, since the feature/doc-actions branch will no longer require manual submission of the GitHub Action.

Reply:
Done!

Reply:
Thanks!